build.ps1

```diff
@@ -1432,6 +1445,7 @@ Install-HostToolchain
 
 if (-not $SkipBuild) {
   Build-Inspect $HostArch
+  Build-Format $HostArch
```

Comment:
It might be worth considering running the swift-format tests in the future in this script as in the toolchain yml.